srv/components/provisioner/config.sls

```diff
@@ -27,3 +27,16 @@ provisioner_rsyslog_conf_updated:
     - makedirs: True
     - watch_in:
       - service: Start rsyslog
+
+# Always start glusterfshsaredstorage before salt-minion
+Update glusterfshsaredstorage.service:
```

Comment:
gluster and salt are configured as part of setup stage, can we move that there, e.g. to [client configuration](https://github.com/Seagate/cortx-prvsnr/blob/pre-cortx-1.0/api/python/provisioner/srv/salt/glusterfs/client/config.sls).

Reply:
@andkononykhin 
I am concerned that after update at some point in future if we update gluster services, we would lose this service customization if it is part of `./api/python`.
Being part of `srv/components/provisioner` would take care of update scenario and ensure the customization is applied every time we do an update.